base/libgit2/remote.jl

```diff
@@ -1,5 +1,10 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+"""
+    GitRemote(repo::GitRepo, rmt_name::AbstractString, rmt_url::AbstractString) -> GitRemote
+
+Look up a remote git repository using its name and url. Uses the default fetch refspec.
```

Comment:
Since URL is an acronym it might be nice to have it in uppercase. One could argue that Git is a proper noun and so it should be in title case, though I'm not sure whether capitalizing it here would be consistent with the existing docstrings.